### PR TITLE
[Doc] correctify description for duplicate key table

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Signed-off-by: hellolilyliuyi <hellolilyliuyi123@163.com>

## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 2.5
  - [x] 2.4
  - [x] 2.3
  - [x] 2.2

### CODE REVIEW

File: docs/table_design/Data_model.md [hunk 1]
```diff
@@ -385,4 +385,4 @@ PROPERTIES("replication_num" = "3",
 
 ### What to do next
 
-You can run a  stream load, broker load, or routine load job to perform insert, update, or delete operations on all or individual columns of a table that uses the Primary Key model. For more information, see [Data load into tables of Primary Key model](../loading/Load_to_Primary_Key_tables.md).
+You can run a  stream load, broker load, or routine load job to perform insert, update, or delete operations on all or individual columns of a table that uses the Primary Key model. For more information, see [Data import](../loading/Loading_intro.md).
```
Comment (evelynzhaojie) [inline docs/table_design/Data_model.md:388]: 文档名称写成 Overview of data loading 这样比较标准

File: docs/table_design/Data_model.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ DISTRIBUTED BY HASH(user_id) BUCKETS 8;
 
 - You can create indexes such as BITMAP indexes and Bloomfilter indexes at table creation.
 
-- If two identical records are loaded, the Duplicate Key model considers the two records as one record instead of two.
+- If two identical records are loaded, the Duplicate Key table retains them as two rows rather than one row.
```
Comment (evelynzhaojie) [inline docs/table_design/Data_model.md:91]: 建议 If two identical records are loaded, the Duplicate Key model retains them as two records, rather than one.

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Comment by hellolilyliuyi** [comment]
run starrocks_admit_test

**Comment by hellolilyliuyi** [comment]
@mergify backport branch-2.4

**Comment by hellolilyliuyi** [comment]
@mergify backport branch-2.3

**Comment by hellolilyliuyi** [comment]
@mergify backport branch-2.3

**Comment by hellolilyliuyi** [comment]
@mergify backport branch-2.2

**Pull request merged**